ci/stage-test-sgx.jenkinsfile

```diff
@@ -58,6 +58,10 @@ stage('test-sgx') {
             sh '''
                 cd Examples/curl
                 sed -i '/@rm OUTPUT/d' Makefile
+                if [ "${gcc_dump_machine}" == 'x86_64-redhat-linux' ]
```

Comment:
instead of adding at some line number which might change, why don't you add it towards the end of the file or the first line?

Reply:
@anjalirai-intel what do you say?

Reply:
Yes, adding it in the end will be best

Reply:
I will add it at the end.